tests/unit/parser.js

```diff
@@ -6935,6 +6935,7 @@ exports.testStrictDirectiveASI = function (test) {
     .test("'use strict'\n(function fn() {})();", options);
 
   TestRun(test, 5)
+    .addError(2, "Missing \"use strict\" statement.")
```

Comment:
What exactly is the behavior of this patch? There's a use strict directive in the source, so i don't understand why it's being warned about?